compiler/rustc_middle/src/query/mod.rs

```diff
@@ -143,11 +143,11 @@ rustc_queries! {
 
     /// Represents crate as a whole (as distinct from the top-level crate module).
     ///
-    /// If you call `hir_crate` (e.g., indirectly by calling `tcx.hir().krate()`),
+    /// If you call `hir_crate` (e.g., indirectly by calling `tcx.hir_crate()`),
     /// we will have to assume that any change means that you need to be recompiled.
     /// This is because the `hir_crate` query gives you access to all other items.
-    /// To avoid this fate, do not call `tcx.hir().krate()`; instead,
-    /// prefer wrappers like `tcx.visit_all_items_in_krate()`.
+    /// To avoid this fate, do not call `tcx.hir_crate()`; instead,
+    /// prefer wrappers like [`tcx.hir_visit_all_item_likes_in_crate()`].
```

Comment:
This doesn't look like the correct syntax?

Reply:
rustdoc will strip the backticks, however, it will **not** strip the parenthesis, those need to be removed.

Reply:
It doesn't strip the parentheses because ``[`f()`]`` links to the same item as `[fn@f]`. However the `tcx.` prevents the link from resolving iinm as rustdoc doesn't perform method resolution for intra-doc links.

Reply:
oh, good catch! it needs to be formatted like an associated function, not like a method call.

Reply:
Fixed, and `()` removed to look more like other doc links in the same file.